<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tcpserver</class>
 <widget class="QWidget" name="tcpserver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tcpserver</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>492</width>
     <height>260</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_save_dir">
        <property name="font">
         <font>
          <family>Adobe 宋体 Std L</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>云盘存储路径：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_save_dir">
        <property name="font">
         <font>
          <family>Adobe 宋体 Std L</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QListWidget" name="listWidget_show_usr_state">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_show_request">
        <property name="font">
         <font>
          <family>Adobe 宋体 Std L</family>
          <pointsize>9</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_ip">
        <property name="font">
         <font>
          <family>Adobe 宋体 Std L</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>IP：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_id">
        <property name="font">
         <font>
          <family>Adobe 宋体 Std L</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_port">
        <property name="font">
         <font>
          <family>Adobe 宋体 Std L</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Port：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_port">
        <property name="font">
         <font>
          <family>Adobe 宋体 Std L</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cahnge_ip_port">
        <property name="text">
         <string>更改Ip和端口和路径</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
